.github/workflows/openai-privacy-filter-live.yml

```
@@ -0,0 +1,111 @@
+concurrency:
+  group: openai-privacy-filter-live-${{ inputs.profile || 'cpu' }}
+  cancel-in-progress: ${{ 1 == 0 }}
+
+jobs:
+  live:
+    if: github.repository == 'EdamAme-x/pentect'
+    runs-on: ${{ inputs.profile == 'cuda' && fromJSON('["self-hosted","linux","x64","gpu","nvidia"]') || 'ubuntu-22.04' }}
+    timeout-minutes: 150
+    env:
+      OPF_PROFILE: ${{ inputs.profile || 'cpu' }}
+      PENTECT_OPF_ROOT: ${{ runner.temp }}/pentect-opf-${{ inputs.profile || 'cpu' }}
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)